File: service-batch/src/main/java/com/issuemoa/batch/domain/subsidy/Subsidy.java
package com.issuemoa.batch.domain.subsidy;

import lombok.*;

import javax.persistence.*;
import java.time.LocalDateTime;

@ToString
@AllArgsConstructor
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@Getter
@Builder
@Table(name = "subsidy")
@Entity
public class Subsidy {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String registrationDatetime ;
    private String departmentName;
    private String userType;
    private String detailViewUrl;
    private String serviceId;
    private String serviceName;
    private String servicePurposeSummary;
    private String serviceCategory;
    private String eligibilityCriteria;
    private String responsibleAgencyName;
    private String responsibleAgencyType;
    private String responsibleAgencyCode;
    private String lastModifiedDatetime;
    private String applicationPeriod;
    private String applicationMethod;
    private String contactInfo;
    private String receivingAgency;
    private String viewCount;
    private String supportDetails;
    private String eligibleRecipients;
    private String supportType;
    private Long registerId;
    private Long modifyId;
    private LocalDateTime registerTime;
    private LocalDateTime modifyTime;
}

File: service-subsidy/src/main/kotlin/com/issuemoa/subsidy/entity/Subsidy.kt
package com.issuemoa.subsidy.entity

import java.time.LocalDateTime
import javax.persistence.*

@Entity
@Table(name = "subsidy")
data class Subsidy(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long = 0,
    val registrationDatetime: String? = null,
    val departmentName: String? = null,
    val userType: String? = null,
    val detailViewUrl: String? = null,
    val serviceId: String? = null,
    val serviceName: String? = null,
    val servicePurposeSummary: String? = null,
    val serviceCategory: String? = null,
    val eligibilityCriteria: String? = null,
    val responsibleAgencyName: String? = null,
    val responsibleAgencyType: String? = null,
    val responsibleAgencyCode: String? = null,
    val lastModifiedDatetime: String? = null,
    val applicationPeriod: String? = null,
    val applicationMethod: String? = null,
    val contactInfo: String? = null,
    val receivingAgency: String? = null,
    val viewCount: String? = null,
    val supportDetails: String? = null,
    val eligibleRecipients: String? = null,
    val supportType: String? = null,
    val registerId: Long? = null,
    val modifyId: Long? = null,
    val registerTime: LocalDateTime? = null,
    val modifyTime: LocalDateTime? = null
)